src/azure-cli/azure/cli/command_modules/eventhubs/Operation/Namespace_custom_file.py

```diff
@@ -0,0 +1,227 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+def create_eventhub_namespace(cmd, resource_group_name, namespace_name, location=None, tags=None, sku='Standard',
+                         capacity=None,  tier='Standard', mi_user_assigned=None, mi_system_assigned=False,
+                          encryption_config=None, minimum_tls_version=None,disable_local_auth = None,
+                         maximum_throughput_units=None,private_endpoint_connection_name='enabled',
+                         require_infrastructure_encryption=None,is_kafka_enabled=None,
+                              is_auto_inflate_enabled=None,alternate_name=None, zone_redundant=None):
+    from azure.cli.command_modules.eventhubs.aaz.latest.eventhubs.namespace import Create
+    user_assign = {}
+    dict1 = {}
+    a = "None"
+    print(encryption_config)
+    dict1 = {
+        "resource_group": resource_group_name,
+        "namespace_name": namespace_name,
+        "tags": tags,
+        "sku": {
+            "name": sku,
+            "capacity": capacity,
+            "tier": sku
+        },
+        "minimum_tls_version": minimum_tls_version,
+        "location": location,
+        "enable_auto_inflate": is_auto_inflate_enabled,
+        "maximum_throughput_units":maximum_throughput_units,
+        "is_kafka_enabled":is_kafka_enabled,
+        "zone_redundant": zone_redundant,
+        "private_endpoint_connection_name": private_endpoint_connection_name,
```

Comment:
There is no variable in namespace API by the name private_endpoint_connection_name, you probably mean public_network_access